test/language_server_test.rb

```diff
@@ -319,89 +319,6 @@ def test_document_links_from_correct_root
     })
   end
 
-  def test_sends_empty_diagnostic_for_fixed_offenses
```

Comment:
This test was removed because it is redundant and covered by the DiagnosticsEngine unit tests.